WhatsApp/app/build.gradle

```diff
@@ -64,10 +66,19 @@ dependencies {
     implementation 'androidx.navigation:navigation-fragment-ktx:2.4.1'
     implementation 'androidx.navigation:navigation-ui-ktx:2.4.1'
     implementation 'androidx.core:core-ktx:1.7.0'
+    implementation "androidx.cardview:cardview:1.0.0"
     implementation 'androidx.appcompat:appcompat:1.4.1'
     implementation 'com.google.android.material:material:1.5.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+
+    //Experimental coroutines
+    kotlin{
+        experimental{
+            coroutines "enable"
+        }
+    }
```

Comment:
is this necessary?

Reply:
was just testing some things not really necessary can remove it